### PR TITLE
Use std mp

### PR DESCRIPTION
**Pull request by hellmanj**

### CODE REVIEW

File: deflake/deflake.py [hunk 1]
```diff
@@ -22,6 +24,42 @@ def out(self, txt, color=None):
         print(out)
 
 
+def init_worker():
```
Comment (kahunacohen) [inline deflake/deflake.py]: Does this want to be "protected" with a leading underscore?

**Comment by kahunacohen** [comment]
More nice work! :+1:

**Comment by hellmanj** [comment]
Thanks!

**Comment by kahunacohen** [comment]
Sweet. Are you able to deploy to pypi?

**Comment by hellmanj** [comment]
Just did, version 0.1.0

**Pull request merged**